nubis/puppet/00_nat.pp

```diff
@@ -26,3 +26,7 @@
     nat_in_interface  => 'eth1',
     nat_out_interface => 'eth0',
 }
+
+class { 'nsm':
+    nsm_mailto  => 'michal@mozilla.com'
```

Comment:
This should go to a distribution list, not to an individual email address. That way infosec can control their own notifications and we will not be required to update every account when the responsible person changes.

Also it might be worth considering generalizing this. Meaning having it as an input parameter to the build pipeline or updating it at run-time through consul. This helps to make this stuff more portable and reusable.